wiki/Dev Environment MAMP.md

```diff
@@ -1,126 +1,79 @@
 # CAJA Development Environment Setup
 How to set up, run, and debug a production clone.
-Written for OSX
+Written for Mac OS
 
 ## Prerequsites:
-- wkhtmltopdf used to generate assembled documents, easiest to install using Homebrew: http://brewformulas.org/Wkhtmltopdf
-- node.js version v12.5.0
-- npm 6.9.0
-- MAMP
+
+- wkhtmltopdf used to generate assembled documents, download and install instructions https://wkhtmltopdf.org/downloads.html
+- NOTE: refer to https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac for getting around the `unidentified developer` warning if it pops up for you.
+- node.js version v12.20.1
+- npm 6.14.10
+- MAMP 5.7 Install via [Homebrew](https://brew.sh/) or direct [download](https://www.mamp.info/en/downloads/)
     - First kill system apache: `sudo apachectl stop`
+    - Open MAMP and go to `preferences`, check the Ports, PHP and Web Server tabs for the below settings to match.
+    - PHP v 7.4.2
```

Comment:
This should actually be PHP 8+ but drupal 7 has issues with php 8 so this will not block this particular merge.

Reply:
Note, not issue